ompi/mca/coll/base/coll_base_util.c

```diff
@@ -569,6 +570,30 @@ int mca_coll_base_name_to_colltype(const char* name)
     return -1;
 }
 
+/**
+ * topo level conversions both ways; str <-> id
+ * An enum is used for conversions.
+ */
+static mca_base_var_enum_value_t level_enumerator[] = {
+    { INTRA_NODE, "intra_node" },
+    { INTER_NODE, "inter_node" },
+    { GLOBAL_COMMUNICATOR, "global_communicator" },
```

Comment:
I'm terrible at naming things but I generally prefer shorter, e.g. `internode`, `intranode`, `global`.

Reply:
I used the same name as in HAN for consistency
https://github.com/open-mpi/ompi/blob/main/ompi/mca/coll/han/coll_han_component.c#L179-L184